src/components/Calculator.js

```diff
@@ -6,45 +11,58 @@ class Calculator extends Component {
   // eslint-disable-next-line no-useless-constructor
   constructor(props) {
```

Comment:
- [ ] Same here, Please remove all the `eslint `disablement lines and fix eslint error in your code.

Reply:
Fixed.